gasstation/gasstattion_test.go

```diff
@@ -46,7 +46,7 @@ func TestSuggestGasPriceForUserAction(t *testing.T) {
 	blkState := blockchain.InMemStateFactoryOption()
 	blkMemDao := blockchain.InMemDaoOption()
 	blkRegistryOption := blockchain.RegistryOption(&registry)
-	bc := blockchain.NewBlockchain(cfg, blkState, blkMemDao, blkRegistryOption)
+	bc := blockchain.NewBlockchain(cfg, nil, blkState, blkMemDao, blkRegistryOption)
```

Comment:
instead of using blkMemDao, we could generate a blockMemDao, and then use it as the second parameter.

Reply:
fixed in latest push